src/operations/blas3/gemm_no_local.hpp

```diff
@@ -93,26 +91,14 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, tile_type,
   output_t c_;
   element_t alpha_;
```

Comment:
same as above alpha and beta can be const

Reply:
Fixed!